### ISSUE TITLE
Incompatible with Vuetify

### ISSUE THREAD

**Issue by scscgit** [danielroe/nuxt-vitest#151]
I can't find a valid combination to use with Vuetify. I created an issue at `@nuxt-alt/vuetify`, where the `vite-plugin-vuetify` causes error `Vuetify plugin must be loaded after the vue plugin`:
* https://github.com/nuxt-alt/vuetify/issues/9

This is caused by `vite:extendConfig` Nuxt hook not having the `vite:vue` Plugin accessible, so it can't reorder it. (Using `vite:configResolved` hook instead didn't work either.)

Note that this issue only occurs when `vitest.config.mjs` is used, so I assume there may be some `nuxt-vitest`-specific zero-config Vite misconfiguration. I'd appreciate any hints about the root cause (or alternatives).

**Comment by danielroe** [comment]
Could you provide a reproduction? 🙏

**Comment by scscgit** [comment]
Sure, here is the same content from the linked issue:

* Install dependencies:
`"nuxt": "^3.4.3",`
`"@nuxt-alt/vuetify": "^1.1.1",`
`"`[nuxt-vitest](https://github.com/danielroe/nuxt-vitest)`": "^0.6.10",`

* Add both modules to `nuxt.config.ts` (any order):
```ts
export default defineNuxtConfig({
  modules: ['nuxt-vitest','@nuxt-alt/vuetify']
})
```

* Add file `vitest.config.mjs` to the project root directory with the following content:
```js
import { defineVitestConfig } from 'nuxt-vitest'

export default defineVitestConfig({})
```

* (Optional) add any test

* Run `npm run test` with script in `package.json`:
```
"scripts": {
    "test": "vitest",
  },
```
Following error occurs:
```cmd
> vitest
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Unhandled Error ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Error: Vuetify plugin must be loaded after the vue plugin
    at configResolved (~\node_modules\vite-plugin-vuetify\dist\importPlugin.js:19:23)
```
(as the `config.plugins.findIndex(plugin => plugin.name === 'vite:vue')` unexpectedly matches the plugin.)

**Comment by danielroe** [comment]
I meant, would you create a runnable reproduction, e.g. from https://stackblitz.com/github/nuxt/starter/tree/v3.

**Comment by danielroe** [comment]
Added a template you can use: https://stackblitz.com/edit/nuxt-vitest

**Comment by scscgit** [comment]
Wow, it seems this issue has been already fixed in version `0.6.11` (my repro referenced `0.6.10`).

Here is a repro using old version with the error: https://stackblitz.com/edit/nuxt-vitest-qahjva?file=package.json

You can thus probably link this issue with the closed PR https://github.com/danielroe/nuxt-vitest/pull/148
